.baseline/checkstyle/checkstyle.xml

```diff
@@ -1,16 +1,9 @@
-<?xml version="1.0"?>
-<!DOCTYPE module PUBLIC
-        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
-        "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
removal of doctype seems to [cause checkstyle task to fail](https://app.circleci.com/pipelines/github/palantir/tritium/965/workflows/3d66e3a6-cdf6-49ac-8ebb-3208e277b00e/jobs/11971).